lein-cljfmt/src/leiningen/cljfmt.clj

```diff
@@ -57,8 +57,12 @@
 
 (defn check
   ([project]
-   (apply check project (format-paths project)))
+   (if project
+     (apply check project (format-paths project))
+     (main/abort "No project found and no source paths provided")))
   ([project path & paths]
+   (when-not project
+     (main/info "Warning: project is nil, assuming . is project root"))
```

Comment:
Instead of `main/info`, perhaps `main/warn` would be more appropriate.

However, do we need this warning? I don't think we need to print a warning message if the user isn't using the plugin in an unexpected way. Is there a reason we'd want to keep the warning?